libraries/streaming-e2e-tests/build/create-react-app.yml

```diff
@@ -5,28 +5,18 @@ steps:
     displayName: 'create-react-app'
     workingDirectory: $(RootReactPiecesDir)
   
-  - powershell: npm install botframework-webchat@4.9.2
-    displayName: 'npm install botframework-webchat'
-    workingDirectory: $(RootReactPiecesDir)/react-app
-
-  - powershell: Remove-Item src/App.js, src/index.js, public/index.html
-    displayName: "Delete files we're replacing with custom code in react app"
-    workingDirectory: $(RootReactPiecesDir)/react-app
+  - powershell: Remove-Item src/App.js, src/index.js, public/index.html, ./package.json, ./yarn.lock
```

Comment:
lock file, whether it be package-lock or yarn.lock, is needed when using cache, otherwise this sequence happens:
* `npx create-react-app`
* restore Web Chat cache successfully

```
Failed to compile.

.\src\App.js
Cannot find module: 'botframework-webchat'. Make sure this package is installed
```

Even though when viewing node modules, you actually have all of the `botframework-webchat` folders.